.github/workflows/main.yml

```diff
@@ -588,19 +602,20 @@ jobs:
           tar -xvf umamba.tar
 
       - name: create build environment
-        uses: mamba-org/provision-with-micromamba@main
+        uses: mamba-org/setup-micromamba@v1
         with:
           environment-file: ./micromamba/environment-dev.yml
           environment-name: build_env
-          extra-specs: menuinst
-
+          create-args: menuinst
+          init-shell: bash cmd.exe powershell
```

Comment:
Setting `init-shell: none` could also be an option. This would then definitely not interfere with the shell initialization tests...

We would then need to activate the micromamba environment by hand or call `micromamba run -n ...`

Reply:
I think when `init-shell` was not used, the shell was not finding `micromamba` executable

Reply:
Yes, this is because the micromamba executable is not on PATH. You could add it to PATH manually beforehand:

```yml
      - name: Add micromamba to GITHUB_PATH
        run: echo "${HOME}/micromamba-bin" >> "$GITHUB_PATH"
```

Reply:
Maybe I lobby for https://github.com/mamba-org/mamba/issues/2577

Reply:
> provision-with-micromamba did the shell initialization thing for every shell that was on the system. setup-micromamba allows you to specify the shell and also gives you the option to do the initialization for no shell at all. I think this would be the option that makes the most sense for shell init tests.

I agree this would be better, but I'm also afraid by potential issues this could lead to and the time it would take to fix them. Let's merge this one since it passes the, and and tackle this in a dedicated PR.